src/suit/js/SUIT.js

```diff
@@ -157,12 +124,16 @@ let options = {
     hips: {
         readoutShowsPixel : true,
         hipsSources: 'lsst,cds',
-        defHipsSources: {source: 'lsst', label: 'Featured'},
+        defHipsSources: {source: 'lsst', label: 'Rubin Featured'},
+        mergedListPriority: 'lsst',
         adhocMocSource: {
-            sources: getDefaultMOCList(),
-            label: 'Featured MOC '
+            sources: [
+                'temp://lsst/dp02_dc2/hips/images/color_gri',
+                'temp://lsst/dp02_dc2/hips/images/band_u',
+                'temp://lsst/dp02_dc2/hips/images/band_g',
```

Comment:
This should be the whole list of 7 (gri,u,g,r,i,z,y).  At a later date we should figure out how to make this more metadata-configurable.

Reply:
I use a sub list to better indicate when the whole list was not picked up from the portal/templates/deployment. We should never be relying on this list. It should always be overridden.

Reply:
That makes sense; so I'll take a look at what in phalanx for this.